steps/wellbeing-support/support-needs.tsx

```diff
@@ -0,0 +1,423 @@
+          // eslint-disable-next-line react/display-name
+          Component: (props): JSX.Element => {
+            const summary = {
+              value: (
+                <>
+                  <h2>Children&apos;s and Young People&apos;s safeguarding</h2>
+                  <br />
+                  <span>
+                    Have you or a member of the household expressed concerns
+                    relating to children and young people’s safeguarding?
+                  </span>
+                </>
+              ),
+              id: "childrens-safeguarding-summary"
+            };
+            return (
+              <Details summary={summary}>
+                <TextArea {...props}></TextArea>
+                <Paragraph>Or refer now:</Paragraph>
+                <Paragraph>
+                  Hackney First Access Screening Team (FAST)
+                  <br />
+                  Email:{" "}
+                  <Link href="mailto:fast@hackney.gov.uk">
+                    fast@hackney.gov.uk
+                  </Link>
+                  <br />
+                  Phone: 020 8356 5500
+                </Paragraph>
+                <Paragraph>
+                  Children&apos;s and young people&apos;s safeguarding
+                  <br />
+                  Email:{" "}
+                  <Link href="mailto:cscreferrals@hackney.gov.uk">
+                    cscreferrals@hackney.gov.uk
+                  </Link>
+                  <br />
+                  Phone: 020 8356 4844
+                  <br />
+                  Out of hours: 020 8356 2710
+                </Paragraph>
+              </Details>
+            );
+          },
+          props: {
+            name: "childrens-safeguarding-notes",
+            label: {
+              value: "Add note for post visit referral."
+            }
+          },
+          defaultValue: "",
+          emptyValue: "",
+          databaseMap: new ComponentDatabaseMap<
+            ProcessDatabaseSchema,
+            "supportNeeds"
+          >({
+            storeName: "supportNeeds",
+            key: processRef,
+            property: ["childrenYoungPeopleSafeguardingNotes"]
+          })
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "domestic-violence",
+          // eslint-disable-next-line react/display-name
+          Component: (props): JSX.Element => {
+            const summary = {
+              value: (
+                <>
+                  <h2>Domestic or sexual violence</h2>
+                  <br />
+                  <span>
+                    Have you or a member of the household expressed concerns
+                    relating to domestic or sexual violence?
+                  </span>
+                </>
+              ),
+              id: "domestic-violence-summary"
+            };
+            return (
+              <Details summary={summary}>
+                <TextArea {...props}></TextArea>
```

Comment:
As Chris suggested, I think we probably want to use `TextAreaDetails` for this, too, and to add optional `contentBeforeTextArea` and `contentAfterTextArea` props.